libr/util/sys.c

```diff
@@ -255,8 +262,10 @@ R_API int r_sys_sleep(int secs) {
 
 R_API int r_sys_usleep(int usecs) {
 #if __UNIX__ || __CYGWIN__ && !defined(MINGW32)
-	// unix api uses microseconds
-	return usleep (usecs);
+	struct timespec rqtp;
+	rqtp.tv_sec = usecs / 1000000;
+	rqtp.tv_nsec = (usecs % 1000000) * 1000;
```

Comment:
What about reusing division result and use substract instead of doing modulo here? Should be faster. Not sure if compilers will optimize this